modules/adagioRtdProvider.js

```diff
@@ -31,6 +31,8 @@ import { _ADAGIO, getBestWindowForAdagio } from '../libraries/adagioUtils/adagio
 import { getGptSlotInfoForAdUnitCode } from '../libraries/gptUtils/gptUtils.js';
 import { getBoundingClientRect } from '../libraries/boundingClientRect/boundingClientRect.js';
 
+import {getGlobalVarName} from '../src/buildOptions.js';
```

Comment:
how is this different than https://github.com/prebid/Prebid.js/blob/7f61966262fb08006e6b15094d643ab6cbf794a1/modules/medianetBidAdapter.js#L23

Reply:
getGlobal returns the global object (the one that has `.requestBids()` etc); `getGlobalVarName` returns the global name (e.g. `'pbjs'`).